MinecraftClient/Protocol/Handlers/Protocol18.cs

```diff
@@ -724,7 +724,7 @@ internal bool HandlePacket(int packetID, Queue<byte> packetData)
             }
             catch (Exception innerException)
             {
-                if (innerException is SocketException || innerException.InnerException is SocketException || innerException is ThreadAbortException)
+                if (innerException is SocketException || innerException.InnerException is SocketException)
```

Comment:
Seems like you accidentally reverted this line from commit 4362de4540efe25a56e72cb5db20b4ad7f2a4f2d